### ISSUE TITLE
Push editor syntax highlighting to vim and neovim

### ISSUE THREAD

**Issue by radare** [vlang/v#10321]
There's VLANG support plugin for Vim in

* https://github.com/cheap-glitch/vim-v/
* https://github.com/ollykel/v-vim

But ideally I would prefer not to install any plugin to use V, So i'm filling this issue to track the status of V on 

* [ ] NeoVim https://github.com/neovim/neovim/tree/master/runtime/syntax
* [ ] Vim https://github.com/vim/vim

What about other editors? Emacs?

**Comment by Larpon** [comment]
I'm maintaining https://github.com/Larpon/kate-syntax-highlight-v

**Comment by Cons-Cat** [comment]
I don't think it makes sense to over-centralize the V tooling. I only care about one editor for writing V, so all the others that would be provided are bloat. And I reckon that most other devs also only care about one or two editors for V. Plus, over-centralization leads to friction in developing these plugins. 

If someone wants to write a pull request for a Vim plugin, then they would need to get reviewed by a V developer. How many V devs can confidently review Elisp / Vimscript / Lua / Typescript? Iirc there is at least one dev currently for each of those, but it seems unreasonable imho to make _that one Emacs user_ with write access to this repository single handedly review every Emacs pull request. How long would it take them to get around to these reviews _on top_ of the compiler pr reviews they already have to do? And what happens if they stop working on V, or take a break?

I think the Emacs community can better maintain Emacs packages, and the Neovim community can better maintain Neovim plugins, etc. It's not even that inconvenient to get these V tools set up.

**Comment by Miguevrgo** [comment]
Any advances on this topic?
